src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -285,68 +290,88 @@ def get(self, **kwargs):
         mc_pileup = kwargs.get("mc_pileup", [])
         data_pileup = kwargs.get("data_pileup", [])
         requestor = kwargs.get("requestor", [])
-        mask = kwargs.get("mask", [])
-        detail = kwargs.get("detail", True)
-        # set the return format. default format has request name as a key
-        # if is set to one it returns list of dictionary with RequestName field.
-        common_dict = int(kwargs.get("common_dict", 0))
+
+        # further tweaks to the couch queries
+        if len(status) == 1 and status[0] == "ACTIVE":
+            status = ACTIVE_STATUS
         if detail in (False, "false", "False", "FALSE"):
             option = {"include_docs": False}
         else:
             option = {"include_docs": True}
-        # eventhing should be stale view. this only needs for test
-        _nostale = kwargs.get("_nostale", False)
-        if _nostale:
+        # everything should be stale view. this only needs for test
+        if nostale:
```

Comment:
Are we sure we want to introduce a discrepancy between the key name in the document and the key name we use in the code

Reply:
That's really just a flag, not used for anything else.